agir/lib/model_fields.py

```diff
@@ -139,26 +143,33 @@ class FacebookEventField(models.CharField):
         r"^((?:https://)?(?:www\.)?(?:facebook|fb|m\.facebook)\.(?:com|me)/(?:events|e)/(?:\d\w{0,20}))(?:/.*)?$"
     )
 
+    # First we try to match an URL with an FB event ID (for backward compatibility)
+    match = FACEBOOK_EVENT_ID_RE.match(url)
+    if match:
+        return f"https://www.facebook.com/events/{match.group(1)}"
+    # If no FB id is found, we try to match a supported FB event URL (e.g. to allow for short URLs)
+    match = FACEBOOK_EVENT_URL_RE.match(url)
+    if match:
+        return match.group(1)
+
+    return False
```

Comment:
Je crois que la convention c'est plutôt d'avoir une exception dans le validateur, plutôt que de renvoyer un boolean.
```suggestion
  raise TypeError(INVALID_FACEBOOK_EVENT_LINK_MESSAGE)
```